composeApp/src/commonMain/kotlin/common/BottomBar.kt

```diff
@@ -0,0 +1,115 @@
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.unit.dp
+import dev.icerock.moko.resources.compose.painterResource
+import instaclone.resources.MR
+
+@Composable
+fun BottomBar() {
+    val homeTabState = remember {
+        mutableStateOf(true)
+    }
+    val searchTabState = remember {
+        mutableStateOf(false)
+    }
+    val favoriteTabState = remember {
+        mutableStateOf(false)
+    }
+    val profileTabState = remember {
+        mutableStateOf(false)
+    }
+    val profileBorderColor = remember {
+        mutableStateOf(Color.Transparent)
+    }
+    val tabList = listOf(homeTabState, searchTabState, favoriteTabState, profileTabState)
+
+    val iconModifier = Modifier
+        .width(22.dp)
+        .height(24.dp)
+    Row(
+        modifier = Modifier.background(Color.White).fillMaxWidth().height(45.dp).border(
+            BorderStroke(width = 0.5.dp, color = Color(0x1A000000))
+        ),
+        verticalAlignment = Alignment.CenterVertically,
+        horizontalArrangement = Arrangement.SpaceAround
+    ) {
+        IconButton(onClick = {
+            selectSingleTab(homeTabState, tabList)
+        }) {
+            Icon(
+                painterResource(if (homeTabState.value) MR.images.home_selected else MR.images.home_unselected),
+                modifier = iconModifier,
+                contentDescription = "Home Menu"
+            )
+        }
+        IconButton(onClick = {
+            selectSingleTab(searchTabState, tabList)
+        }) {
+            Icon(
+                painterResource(
+                    if (searchTabState.value) MR.images.search_selected else MR.images.search_unselected
+                ),
+                modifier = iconModifier,
+                contentDescription = "Search Menu"
+            )
+        }
+        IconButton(onClick = {
+        }) {
+            Icon(
+                painterResource(MR.images.add_new),
+                modifier = iconModifier,
+                contentDescription = "Add Menu"
+            )
+        }
+        IconButton(onClick = {
+            selectSingleTab(favoriteTabState, tabList)
+        }) {
+            Icon(
+                painterResource(if (favoriteTabState.value) MR.images.favorite_selected else MR.images.favorite_unselected),
+                modifier = iconModifier,
+                contentDescription = "Favorite Icon"
+            )
+        }
+        IconButton(modifier = Modifier
+            .border(1.dp, profileBorderColor.value, RoundedCornerShape(100)).size(27.dp), onClick = {
+            selectSingleTab(profileTabState, tabList)
+        }) {
+            profileBorderColor.value = if (profileTabState.value) Color.Black else Color.Transparent
+            /*TODO ("Replace with dynamic obtained image from user")*/
```

Comment:
Can we create an issue for that?

Reply:
Hopefully, but I'm guessing API integration of any sort is far down the line